packages/ui/src/components/editor/input/Euler/index.tsx

```diff
@@ -43,66 +43,71 @@ type EulerInputProps = {
   unit?: string
 }
 
+const temp_Euler = new Euler()
+const temp_Quat = new Quaternion()
```

Comment:
we should only use camelCase in ts files

```suggestion
const tempEuler = new Euler()
const tempQuat = new Quaternion()
```